eng/pipelines/templates/steps/npm-release-task.yml

```diff
@@ -16,10 +16,14 @@ steps:
         tar -xzf $tarFile -C $tempDir
         $packageJsonDir = "$tempDir\package\package.json"
         $pkg = Get-Content -Raw "$packageJsonDir" | ConvertFrom-Json
-        $packageName = $pkg.Name
+        # $packageName = $pkg.Name
+        $packageName = "testestestestetsetset"
         $packageVersion = $pkg.Version
-        $packageProps = npm view $packageName -json | ConvertFrom-Json
-        $originalTags = $packageProps.'dist-tags' | ConvertTo-Json -Compress
+        $packageProps = npm view $packageName -json -silent | ConvertFrom-Json
+        $originalTags = "{}"
+        if (!$packageProps.error) {
```

Comment:
Should we verify that `error.code` is 404 specifically? For other failures (e.g. npm api, etc.) do we want to continue or fail?